scripts/dream.py

```diff
@@ -14,12 +14,30 @@
 from ldm.dream.server import DreamServer, ThreadingDreamServer
 from ldm.dream.image_util import make_grid
 from omegaconf import OmegaConf
+import torch
 
 # Placeholder to be replaced with proper class that tracks the
 # outputs and associates with the prompt that generated them.
 # Just want to get the formatting look right for now.
 output_cntr = 0
 
+def fix_func(orig):
```

Comment:
It feels like this chunk of code should be moved into ldm.generate. Otherwise people who are writing scripts against Generate won't get the benefits of your fix.